policies/policies.go

```diff
@@ -39,7 +39,7 @@ var (
 	log = logrus.WithField("component", "policies")
 )
 
-type metricsResourceTypeCache struct {
+type metricsToolIDAndResourceTypeCache struct {
```

Comment:
I like that name :)

Reply:
I would prefer just metricsCache and write a comment whats the key of. I didn't want to change it though. Maybe I will later :D